packages/debugger/src/panels/breakpoints/body.tsx

```diff
@@ -135,7 +155,21 @@ const BreakpointComponent = ({
       onClick={(): void => model.clicked.emit(breakpoint)}
       title={breakpoint.source?.path}
     >
-      <span className={'jp-DebuggerBreakpoint-marker'}>●</span>
+      {!isSelected ? (
+        <span
+          className={'jp-DebuggerBreakpoint-marker'}
+          aria-label="Breakpoint"
```

Comment:
The labels would ideally be translated.

```suggestion
          aria-label={trans.__('Breakpoint')}
```

where `trans` needs to be threaded down through components.